core/server/data/migration/update.js

```diff
@@ -63,18 +88,16 @@ update = function update(fromVersion, toVersion, logger) {
     // This shouldn't include the from/current version (which we're already on)
     var versionsToUpdate = versioning.getMigrationVersions(fromVersion, toVersion).slice(1);
 
-    return backup(logger).then(function () {
-        return updateDatabaseSchema(versionsToUpdate, logger);
-    }).then(function () {
-        // Ensure all of the current default settings are created (these are fixtures, so should be inserted first)
-        return fixtures.ensureDefaultSettings(logger);
-    }).then(function () {
-        // Next, run any updates to the fixtures, including default settings, that are required
-        return fixtures.update(versionsToUpdate, logger);
-    }).then(function () {
-        // Finally update the database's current version
-        return versioning.setDatabaseVersion();
-    });
+    return backup(logger)
+        .then(function () {
+            return Promise.mapSeries(versionsToUpdate, function (versionToUpdate) {
+                return migrateToDatabaseVersion(versionToUpdate, logger, modelOptions);
+            });
+        })
+        .catch(function () {
+            // we don't want that your blog can't start
```

Comment:
this behaviour is changed in #7019 , because we discussed in `slack` that running a code base with a different database version can cause trouble.